framework/src/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -505,6 +518,29 @@ private void parseMethod(MethodDeclaration decl, ExecutableElement elt,
         putNew(atypes, elt, methodType);
     }
 
+    private void handleExistingAnnotations(AnnotatedTypeMirror atype, Type typeDef) {
```

Comment:
I added Javadoc and a short paragraph in the manual. Please have a look through the manual change.